### ISSUE TITLE
When there are many spans in a trace, the trace display shows an anomaly.

### ISSUE THREAD

**Issue by xuwei6688** [SigNoz/signoz#2523]
## Bug description

When there are a large number of spans in a trace, trace display always has problems, such as not displaying spans, page layout becoming chaotic, and so on.

## Expected behavior
![image](https://user-images.githubusercontent.com/31433665/228802031-0fd3fea9-30b4-43b0-95b9-f9257313eee9.png)

another case
![image](https://user-images.githubusercontent.com/31433665/228802636-fa7b6f83-dbb0-4529-8dec-ec6de984a553.png)

## How to reproduce
1. version is v0.16.2.
2.There are a large number of spans in the same trace, such as 150.

## Version information 
* **Signoz version**: version is v0.16.2.
* **Browser version**:  chrome 111.0.5563.110 (arm64)
* **Your OS and version**: centos7
* **Your CPU Architecture**(ARM/Intel):  intel

**Comment by xuwei6688** [comment]
> We have seen many users with even 1000s of spans in a trace without any issue. Can you dump api response of this page either here or DM in our community slack after removing sensitive data? We should look into this

Of course, And I think I found the reason: the depth of nested spans is too deep.

**Comment by xuwei6688** [comment]
I simulated this situation and found that it occurs when the depth reaches over 100.

![image](https://user-images.githubusercontent.com/31433665/229106948-627acd7d-02fe-4d99-9987-ec62c0146092.png)

**Comment by scaler-tauseef** [comment]
This is still an issue, In my case sometimes there are 1500+ spans in a trace. When I open that trace the whole UI is stuck and I need to kill the browser tab

**Comment by srikanthccv** [comment]
not an issue anymore
